Brokerages/Binance/BinanceOrderBook.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuantConnect.Brokerages.Binance
+{
+    /// <summary>
+    /// Represents a full order book for a security.
+    /// It contains prices and order sizes for each bid and ask level.
+    /// The best bid and ask prices are also kept up to date.
+    /// </summary>
+    public class BinanceOrderBook: DefaultOrderBook
```

Comment:
Are these intended to be completely self-contained/auto-updating or simply data structures?

@StefanoRaggi @Martin-Molinero 
I was thinking it would be nice if order books were self contained, such that they manage their own auto-update, etc. Thoughts?